lightning/src/ln/functional_test_utils.rs

```diff
@@ -357,7 +357,7 @@ impl<'a, 'b, 'c> Drop for Node<'a, 'b, 'c> {
 			}
 
 			// Check that if we serialize the Router, we can deserialize it again.
```

Comment:
Hmm.. maybe we should update this to say `NetworkGraph` now that we've come full circle and have a `Router` type again lol.